src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const MALE = 'm';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+      {
+        isMarried
+          ? (
+            <p className="Person__partner">
+              {
+                `${partnerName} is my ${sex === MALE
+                  ? 'wife'
+                  : 'husband'}`
+              }
+            </p>
+          )
+
+          : <p className="Person__partner">I am not married</p>
+      }
+    </section>
```

Comment:
Can we check `isMarried` inside our `<p>` to avoid repeating tag?

Would be really cool to create a separate variable for `sex === MALE ? 'wife' : 'husband'`



```suggestion
            <section className="Person">
              <h2 className="Person__name">{`My name is ${name}`}</h2>

              {age && <p className="Person__age">{`I am ${age}`}</p>}

              {isMarried ? (
                <p className="Person__partner">
                  {`${partnerName} is my ${sex === MALE ? 'wife' : 'husband'}`}
                </p>
              ) : (
                <p className="Person__partner">I am not married</p>
              )}
            </section>
```